src/lib/EventPublisher.js

```diff
@@ -140,25 +151,30 @@ class EventPublisher extends EventEmitter {
           timeoutHandler();
         }
       }, subscriptionTimeout);
-
-      this.once(eventNamePrefix, (): void => clearTimeout(timeout));
+      this.once(eventNamePrefix, () => {
+        clearTimeout(timeout);
+      });
     }
 
     if (once) {
       this.once(eventNamePrefix, listener);
+      this.once(eventNamePrefix, () => {
+        this._subscriptionsByID.delete(subscriptionID);
```

Comment:
Ok, cause of these lint errors i cant commit at all. 

so that's all for today :(